src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
+const populationsAll = document.body.children[0].children[1].children;
```

Comment:
You can use querySelectorAll here instead
https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll